src/components/pages/doc/mobile-nav/mobile-nav.jsx

```diff
@@ -1,50 +1,85 @@
 import clsx from 'clsx';
-import { navigate } from 'gatsby';
+import { motion, useAnimation } from 'framer-motion';
 import PropTypes from 'prop-types';
-import React, { Fragment } from 'react';
+import React, { useEffect, useState } from 'react';
+import { useWindowSize } from 'react-use';
 
-import { DOCS_BASE_PATH } from 'constants/docs';
+import Item from 'components/pages/doc/sidebar/item';
 import ChevronRight from 'icons/chevron-right.inline.svg';
 
+const ANIMATION_DURATION = 0.2;
+
+const variants = {
+  from: {
+    opacity: 0,
+    translateY: 10,
+    transition: {
+      duration: ANIMATION_DURATION,
+    },
+    transitionEnd: {
+      zIndex: -1,
+    },
+  },
+  to: {
+    zIndex: 20,
+    opacity: 1,
+    translateY: 0,
+    transition: {
+      duration: ANIMATION_DURATION,
+    },
+  },
+};
+
 const MobileNav = ({ className, sidebar, currentSlug }) => {
-  const handleChange = (e) => {
-    const { value, selectedOptions } = e.target;
+  const [isOpen, setIsOpen] = useState(false);
+  const [containerHeight, setContainerHeight] = useState(null);
+  const { height } = useWindowSize();
+  const controls = useAnimation();
 
-    const slug = selectedOptions[0].dataset.standalone ? `/${value}` : `${DOCS_BASE_PATH}${value}/`;
-    if (value) navigate(slug);
+  const openMenu = () => {
+    setIsOpen(!isOpen);
   };
```

Comment:
I'd suggest we avoid closure and leverage the prev state.
```suggestion
  const toggleMenu = () => setIsOpen(isOpen => !isOpen);
  };
```